examples/polynomial/sensitivity/experiment.yml

```diff
@@ -0,0 +1,2 @@
+type: sensitivity
+algorithm: oat
```

Comment:
Could be changed to `one-at-the-time`...

Reply:
On a second thought I agreed with the comment and changed the name of the algorithm to `one-at-a-time`